Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButton.prefab

```diff
@@ -1400,6 +1400,38 @@ MeshRenderer:
   m_SortingLayerID: 0
   m_SortingLayer: 0
   m_SortingOrder: 0
+--- !u!1 &8352633413104042190
```

Comment:
One question that has come up with partners and the pressable button is what is the purpose of the "Static Content?" Is that something we can delete while we are working in here? It seems to always be disabled.

![image](https://user-images.githubusercontent.com/13305729/58264363-97cad500-7d32-11e9-9636-af1588363a32.png)

Reply:
We can delete them. Jonathan placed them as a placeholder when the design was not finalized. I'll update them.

Reply:
Awesome! Thank you Yoon.

Reply:
Removed redundant elements. Updated plate names for easier understanding.
![2019-05-23 15_45_30-Unity 2018 3 8f1 Personal - HandInteractionExamples unity - MRTK-Public-Microsof](https://user-images.githubusercontent.com/13754172/58291981-6f63ca80-7d74-11e9-9144-902b740f175c.png)